applications/DemStructuresCouplingApplication/custom_utilities/dem_structures_coupling_utilities.h

```diff
@@ -29,7 +29,7 @@
 #include "../../DEMApplication/custom_conditions/RigidFace.h"
 #include "../../DEMApplication/DEM_application_variables.h"
 #include "dem_structures_coupling_application_variables.h"
-
+#include "../../DEMApplication/custom_elements/spheric_continuum_particle.h"
```

Comment:
if you need DEM-stuffs then you could include it in the CMakeLists then you dont need the `../../DEMApplication`

Reply:
Thanks @philbucher . Fixed.